Sources/ApodiniHTTPProtocol/Information/Authorization.swift

```diff
@@ -17,10 +17,13 @@ public struct Authorization: HTTPInformation {
 
     public let value: Value
 
-
     public var rawValue: String {
         value.rawValue
     }
+    
+    public var sensitive: Bool {
+        true
+    }
```

Comment:
Yep you're absolutely right, that didn't work out as expected. As mentioned, I will add end-to-end tests in a separate PR :)

If we declare the `sensitive` property on `AnyHTTPInformation`, `ApodiniObserve` would have to depend on `ApodiniHTTPProtocol` again, a path we chose not to follow.

Reply:
> Yep you're absolutely right, that didn't work out as expected. As mentioned, I will add end-to-end tests in a separate PR :)

I think it would be enough if you add it onto the exiting InformationTests to add some asserts here and there for the sensitive property(?)

> If we declare the sensitive property on AnyHTTPInformation, ApodiniObserve would have to depend on ApodiniHTTPProtocol again, a path we chose not to follow (since we would need to query after it in ApodiniObserve).

Nah, the `sensitive` property is declared in the `InformationClass` protocol right? And `ApodiniObserve` accesses the `sensitive` property over the `InformationClass` protocol?

As described above, `AnyHTTPInformation` is the type which conforms to `InformationClass`, so adding the `sensitive` property onto that is the way to go(?)

like
```swift
// adding to AnyHTTPInformation
var sensitive: Bool {
  header == Authorization.header
}
```

Reply:
Yep, that makes a lot of sense.
My initial approach was to remove the `sensitive` property from `InformationClass` completely, but if we leave it in there, my initial problem goes away :)